e2e/johtoselvitys_ja_liite_hankkeelle.spec.ts

```diff
@@ -0,0 +1,256 @@
+import { test, expect } from '@playwright/test';
+import {
+  perustaja,
+  vastaava,
+  suorittaja,
+  rakennuttaja,
+  testiData,
+  testiOsoite,
+  helsinkiLogin,
+  hankeName,
+} from './_setup';
+import path from 'path';
+
+test.beforeEach('Helsinki_login', async ({ page }) => {
+  await helsinkiLogin(page);
+});
+
+test('Johtoselvitys ja liite hankkeelle', async ({ page }) => {
+  test.setTimeout(240000);
+  await page.getByLabel('Luo uusi hanke.', { exact: true }).click();
```

Comment:
The steps to create the hanke are the same here and in `uusi_hanke`. The code for the steps could be shared between the tests.

Maybe there could be shared functions for the steps. Maybe filling a single form page with the 'standard' information could be a shared function, so a test can reimplement that part if it uses different information.